### ISSUE TITLE
I want to set a rate to SVT

### ISSUE THREAD

**Issue by christianbrb** [raiden-network/light-client#1654]
## User Story
As Chris I want to set a rate to SVT so that I can receive payments.

## Acceptance criteria
- It should have a default

**Comment by andrevmatos** [comment]
What default would we use? There're different supported networks. `makeDefaultConfig` can set the default for different ones, or this can be passed also through `.env.<mode>` files on dApp, but I can't see how we can decide on this, and also these values can fluctuate, so it may be hard to follow. Receiving is enabled as soon as the `rateToSvt` mapping is not empty (so per node, not per token), although of course MonitoringRequests are sent only if the token is present in the mapping and the converted value is higher than the set `monitoringReward` (which do have a default, 5 SVT).
So, what would be sensible defaults to put on it? Anything not empty just to enable receiving? Actual defaults to enable MS? How?

**Comment by kelsos** [comment]
Env or somehow make a basic ui where the users can set it?

**Comment by christianbrb** [comment]
@andrevmatos Thanks for your remarks. It's a complicated feature. Typically for a price there needs to be an offer, but due to the design of the Monitoring Service there is no offer and no agreement on a price.

In addition there are three points in time, where the SVT rate is relevant:
1. Set the SVT rate
2. Submit the balance proof to the Monitoring Service Room
3. MS submits the balance proof

We could check the SVT rate before 2. but after 2. rate changes are out of our control. 

Therefore I would propose
1. A sensible default with a safety margin looking at historical data
2. The user needs to confirm the default
3. The user has got the possibility to change the default value at any time
4. (optional) The user get's reminded every month to check the rate

If this is too complicated we should deactivate the feature (https://github.com/raiden-network/light-client/issues/1473) and always submit the balance proof with a pre-configured SVT value (like Raiden Python is doing it).

**Comment by christianbrb** [comment]
@andrevmatos `rateToSvt` is per node and not per token?

**Comment by andrevmatos** [comment]
`rateToSvt` is a mapping, which is per node but can contain rates for each separate token. BUT the `noReceive` capability is per node (for now, we could convert it per-token if needed), and currently it's set if the `rateToSvt` mapping is empty (current default), and gets enabled for all tokens if any rate is present. LC can then receive on all tokens, although of course only if present on the mapping, a MS will be sent (after threshold is achieved).

**Comment by christianbrb** [comment]
@andrevmatos Thanks :)

I am currently doing some [historical analysis of the exchange rates](https://docs.google.com/spreadsheets/d/1U1vOtUQdavuEHAXQ33p9WyTuViaHzL4s9Dnd9ejJBhE/edit#gid=569930139).

From a user perspective
- I don't want to pay more for the monitoring service than I receive when closing my channel
- Deterministically hitting this goal is impossible to achieve right now
- The likely hood for a monitoring service to submit my balance proof is higher if the reward is higher.

There are 3 possible outcomes
1. I am paying more than I would receive when closing the channel on my own
2. I am losing more than I would have paid for the monitoring service
3. I am paying exactly the same (very rare potential result)

As a user I would go with No. 1 as I don't expect my partner to be fraudulent. If he is I would rather like to have the monitoring service kicking in than not doing it and wasting all my SVT tokens and transfer tokens.

Therefore I would need to deduct a safety margin from the [historical rates](https://docs.google.com/spreadsheets/d/1U1vOtUQdavuEHAXQ33p9WyTuViaHzL4s9Dnd9ejJBhE/edit#gid=569930139).

My current proposal for the `ratetoSVT` default:
DAI token = 2
WETH token = 500

Is my thought process correct and understandable?

**Comment by christianbrb** [comment]
Please do not comment yet, we'll discuss it in the daily.

**Comment by christianbrb** [comment]
Decided to not implement this now because it is too complicated, not deterministic and therefor not valuable for the user.
